CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # dev
```

Comment:
If you are tagging this as a new release, you can change the version to reflect that.